specification/under_development/output_schemas/No_Labels_And_Ordered.md

```diff
@@ -1 +1,259 @@
+
+Example log for a single shot:
+
+```log
+START
+METADATA\tmetadata1_name_only
+METADATA\tmetadata2_name\tmetadata2_value
+METADATA\tmetadata3_name\tmetadata3_value
+OUTPUT\tTUPLE\t2
+OUTPUT\tARRAY\t4
+OUTPUT\tRESULT\t0
+OUTPUT\tRESULT\t0
+OUTPUT\tRESULT\t0
+OUTPUT\tRESULT\t0
+OUTPUT\tARRAY\t3
+OUTPUT\tBOOL\ttrue
+OUTPUT\tINT\t42
+OUTPUT\tDOUBLE\t3.1415
+END\t0
+```
+
+## Declared Runtime Functions 
+
+Each of these runtime functions are meant as signifiers to the provider as to how raw device results should be collected into the common output format. For simulation or future environments with full runtime support, these functions can be linked to implementations that directly perform the recording of output to the relevant stream. Each of these functions follow the naming pattern `__quantum__rt__*_record_output` where the initial part indicates the type of output to be recorded.  
+
+### Primitive Result Records 
+
+```llvm
+void @__quantum__rt__result_record_output(%Result*, i8*)
+```
+
+Adds a measurement result to the generated output. It produces output records of exactly `"OUTPUT\tRESULT\t0"` or `"OUTPUT\tRESULT\t1"`. The second parameter defines a string label for the result value. Depending on the output schema, the label is included in the output or omitted.
+
+```llvm
+void @__quantum__rt__bool_record_output(i1, i8*)
+```
+
+produces output records of exactly `"OUTPUT\tBOOL\tfalse"` or `"OUTPUT\tBOOL\ttrue"`
+
+```llvm
+void @__quantum__rt__integer_record_output(i64, i8*)
+```
+
+produces output records of the format `"OUTPUT\tINT\tn"` where `n` is the string representation of the integer value, such as `"OUTPUT\tINT\t42"`
+
+```llvm
+void @__quantum__rt__double_record_output(double, i8*) 
+```
+
+produces output records of the format `"OUTPUT\tDOUBLE\td"` where `d` is the string representation of the double value, such as `"OUTPUT\tDOUBLE\t3.14159"`
+
+### Tuple Type Records
+
+```llvm
+void @__quantum__rt__tuple_record_output(i64, i8*)
+```
+
+Inserts a marker in the generated output that indicates the start of a tuple and how many tuple elements it has. It produces output records of exactly `"OUTPUT\tTUPLE\tn"` where `n` is the string representation of the integer value, such as `"OUTPUT\tTUPLE\t4"`. The second parameter defines a string label for the tuple. Depending on the output schema, the label is included in the output or omitted.
+
+### Array Type Records
+
+```llvm
+void @__quantum__rt__array_record_output(i64, i8*)
+```
+
+Inserts a marker in the generated output that indicates the start of an array and how many array elements it has. It produces output records of exactly `"OUTPUT\tARRAY\tn"` where `n` is the string representation of the integer value, such as `"OUTPUT\tARRAY\t4"`. The second parameter defines a string label for the array. Depending on the output schema, the label is included in the output or omitted.
+
+## Examples
```

Comment:
Since this is the non-labeled schema, shouldn't the examples omit the labels and show the output recording calls with `null` as their label argument?